package.json

```diff
@@ -4,11 +4,22 @@
     "description": "Simple HTML5 charts using the canvas element.",
     "version": "3.8.0",
     "license": "MIT",
+    "type": "module",
     "jsdelivr": "dist/chart.min.js",
     "unpkg": "dist/chart.min.js",
-    "main": "dist/chart.js",
+    "main": "dist/chart.cjs",
     "module": "dist/chart.esm.js",
     "types": "types/index.esm.d.ts",
+    "exports": {
+        ".": {
+          "require": "./dist/chart.cjs",
```

Comment:
should we even support CJS for v4 or just go ESM-only?

Reply:
That is a good question.

Reply:
@kurkle Let's make Chart.js ESM-only?

Reply:
> Let's make Chart.js ESM-only?

@LeeLenaleee @etimberg What do you think?

Reply:
When we say esm only do you mean only shipping a module? Or only the built file and the module?

I think a lot of our users do simply embedded chart.min.js so I think we need to keep supporting that

Reply:
I really just meant ESM-only for Node users. We'd still keep the UMD chart.min.js, but just drop the CJS version

Reply:
@etimberg Good point. We can provide ESM and UMD bandles.